cfep-08.md

```diff
@@ -0,0 +1,329 @@
+ 'pytest',
+ 'pytest-runner',
+ 'python',
+ 'readline',
+ 'rhash',
+ 'setuptools',
+ 'setuptools_scm',
+ 'six',
+ 'sqlite',
+ 'texinfo',
+ 'tk',
+ 'toolchain',
+ 'urllib3',
+ 'wheel',
+ 'win_inet_pton',
+ 'wincertstore',
+ 'xz',
+ 'yaml',
+ 'zlib'}
+ ```
+ ##### 2 (5% of the ecosystem)
+ The package has more than 5% of the ecosystem as indirect or direct decedents.
+ As of writing there are 165 packages meeting this criteria:
+  ```python
+{'asn1crypto',
+ 'atomicwrites',
+ 'attrs',
+ 'autoconf',
+ 'automake',
+ 'backports',
+ 'backports.functools_lru_cache',
+ 'backports_abc',
+ 'bison',
+ 'blas',
+ 'bzip2',
+ 'c_compiler_stub',
+ 'ca-certificates',
+ 'cairo',
+ 'cdt_stub',
+ 'certifi',
+ 'cffi',
+ 'chardet',
+ 'click',
+ 'cmake',
+ 'cmake-binary',
+ 'colorama',
+ 'compatible_pin_stub',
+ 'cryptography',
+ 'cryptography-vectors',
+ 'curl',
+ 'cxx_compiler_stub',
+ 'cycler',
+ 'cython',
+ 'dbus',
+ 'decorator',
+ 'enum34',
+ 'expat',
+ 'flex',
+ 'fontconfig',
+ 'fortran_compiler_stub',
+ 'freetype',
+ 'funcsigs',
+ 'functools32',
+ 'gcc',
+ 'gettext',
+ 'glib',
+ 'gmp',
+ 'gobject-introspection',
+ 'gperf',
+ 'graphite2',
+ 'gsl',
+ 'gst-plugins-base',
+ 'gstreamer',
+ 'harfbuzz',
+ 'hdf5',
+ 'icu',
+ 'idna',
+ 'ipaddress',
+ 'jinja2',
+ 'jom',
+ 'jpeg',
+ 'kiwisolver',
+ 'krb5',
+ 'libedit',
+ 'libffi',
+ 'libflac',
+ 'libgcc',
+ 'libgfortran',
+ 'libgfortran-ng',
+ 'libiconv',
+ 'libjpeg-turbo',
+ 'libogg',
+ 'libpng',
+ 'libsndfile',
+ 'libssh2',
+ 'libtiff',
+ 'libtool',
+ 'libuuid',
+ 'libuv',
+ 'libvorbis',
+ 'libxcb',
+ 'libxml2',
+ 'm4',
+ 'make',
+ 'markupsafe',
+ 'matplotlib',
+ 'more-itertools',
+ 'mpfr',
+ 'mpir',
+ 'msinttypes',
+ 'ncurses',
+ 'nose',
+ 'numpy',
+ 'openblas',
+ 'openssl',
+ 'ordereddict',
+ 'pandas',
+ 'pango',
+ 'pcre',
+ 'perl',
+ 'pip',
+ 'pixman',
+ 'pkg-config',
+ 'pkgconfig',
+ 'pluggy',
+ 'pthread-stubs',
+ 'py',
+ 'pycparser',
+ 'pygments',
+ 'pyopenssl',
+ 'pyparsing',
+ 'pyqt',
+ 'pysocks',
+ 'pytest',
+ 'pytest-runner',
+ 'python',
+ 'python-dateutil',
+ 'pytz',
+ 'pyyaml',
+ 'qt',
+ 'r-base',
+ 'r-lattice',
+ 'r-magrittr',
+ 'r-mass',
+ 'r-matrix',
+ 'r-r6',
+ 'r-rcpp',
+ 'readline',
+ 'requests',
+ 'rhash',
+ 'scipy',
+ 'sed',
+ 'setuptools',
+ 'setuptools_scm',
+ 'singledispatch',
+ 'sip',
+ 'six',
+ 'sqlite',
+ 'ssl_match_hostname',
+ 'subprocess32',
+ 'texinfo',
+ 'texlive-core',
+ 'tk',
+ 'toolchain',
+ 'tornado',
+ 'urllib3',
+ 'wheel',
+ 'win_inet_pton',
+ 'wincertstore',
+ 'xcb-proto',
+ 'xorg-inputproto',
+ 'xorg-kbproto',
+ 'xorg-libice',
+ 'xorg-libsm',
+ 'xorg-libx11',
+ 'xorg-libxau',
+ 'xorg-libxdmcp',
+ 'xorg-libxext',
+ 'xorg-libxrender',
+ 'xorg-libxt',
+ 'xorg-renderproto',
+ 'xorg-util-macros',
+ 'xorg-xextproto',
+ 'xorg-xproto',
+ 'xorg-xtrans',
+ 'xz',
+ 'yaml',
+ 'yasm',
+ 'zlib'}
+```
+##### 3 (Systemic Risk)
+The package has been agreed upon by core that the errors in the feedstock would 
+pose a systemic risk. These packages currently include:
+```python
+{'libgdal',
```

Comment:
I would argue that both of these are domain specific, and do not belong under "systemic risk."  They do affect a system of packages, but not the whole of conda-forge.  Should there be categories of systems?

* geospatial
* deep learning
* bio